container/setup/setup-container.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+# test if ES version supplied
+if [ -z $1 ]; then
```

Comment:
Hmm, does this even work ?

I'm probably missing something but in `Dockerfile` if you don't specify a es_version we will set it to `LATEST` ->
```
ARG ES_VERSION=latest
```

So the string won't be empty in that case ?

Reply:
In the docs I am saying the version has to be explicitly provided when running the build and that results it being passed down to setup container. I much prefer naming things explicitly and not relying on code to handle if values are not provided.